### ISSUE TITLE
Use of code

### ISSUE THREAD

**Issue by ebolisa** [roquita/esp32_light_webpanel#1]
Hi, 
Thanks for sharing!!
Could you please tell us how to implement your code on an ESP32?
I'm connecting to my router ok but when I fire the function below, the data it's not shown in the web explorer nor do I see errors in the Thonny's console.
Surely I'm doing it wrong.
TIA

def web_page():
    file = open("../login/login.html", "r")
    html = file.read()
    file.close()
    return html

def server():
    try:
        import usocket as socket    #importing socket
    except:
        import socket
        gc.collect()
        
    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)   #creating socket object
    s.bind(('', 80))
    s.listen(5)

    while True:
      conn, addr = s.accept()
      print('Got a connection from %s' % str(addr))
      request = conn.recv(1024)
      print('Content = %s' % str(request))
      response = web_page()
      conn.send(response)
      conn.close()

UPDATE:
I'm attaching an img to show what's uploaded on the ESP.

![Capt](https://user-images.githubusercontent.com/17126140/120897352-f6b3cc00-c625-11eb-9aed-9db4f9d08876.png)

**Comment by roquita** [comment]
Did you upload the files?. If yes, check the path you are using while reading.

**Comment by ebolisa** [comment]
Yes, you can see the uploaded folders in the uploaded image and the path (../login/login.htm) seems to be correct. The login page is uploaded but it's not visible so I changed `style="visibility: hidden;` to `style="visibility: visible;`. I can now see the login page but, where are the user/password credentials written at?
Thank you.

**Comment by roquita** [comment]
Readme file was added, please check.

**Comment by roquita** [comment]
Theres is no credentials, you must implement the functionality that you need for html elements.

**Comment by ebolisa** [comment]
Thank you.
